customWorldboss.lua

```diff
@@ -731,9 +732,97 @@ Config_bossYellPhase2[7] = "Sssssszzzzzzzzz!"
+Config_bossSpellModifier2bp0[8] = nil      -- not required if nil
+Config_bossSpellModifier2bp1[8] = nil      -- not required if nil
+Config_bossSpellModifier3bp0[8] = 2000     -- damage to main target
+Config_bossSpellModifier3bp1[8] = 2000     -- damage to secondary target
+Config_bossSpellModifier4bp0[8] = nil      -- not required if nil
+Config_bossSpellModifier4bp1[8] = nil      -- not required if nil
+Config_bossSpellModifier5bp0[8] = nil      -- not required if nil
+Config_bossSpellModifier5bp1[8] = nil      -- not required if nil
+Config_bossSpellModifier6bp0[8] = 3000     -- base damage of shadow blast
+Config_bossSpellModifier6bp1[8] = nil      -- not required if nil
+Config_bossSpellModifier7bp0[8] = nil      -- not required if nil
+Config_bossSpellModifier7bp1[8] = nil      -- not required if nil
+Config_bossSpellModifier8bp0[8] = nil      -- not required if nil
+Config_bossSpellModifier8bp1[8] = nil      -- not required if nil
+
+Config_addHealthModifierParty[8] = 0.2     -- modifier to change health for party encounter. Value in the SQL applies for raid
+Config_addsAmount[8] = 6                   -- how many adds will spawn
+
+Config_addSpell1[8] = 38085             -- min range 30m, 1-3rd farthest target within 30m -- Shadow Blast (just damage)
+Config_addSpell2[8] = 36541             -- min range 45m, cast on tank -- Curse of Burning Shadows (deals shadow damage when fire damage is taken)
+Config_addSpell3[8] = nil               -- min range 0m
+Config_addSpell4[8] = 37479             -- cast on the boss Shadow Mend (heals boss for large amount)
+Config_addSpellEnrage[8] = nil          -- Enrage after 300 seconds
+
+Config_addSpellTimer1[8] = 23000        -- This timer applies to Config_addSpell1
+Config_addSpellTimer2[8] = 13000        -- This timer applies to Config_addSpell2
+Config_addSpellTimer3[8] = 7000         -- This timer applies to Config_addSpell3
+Config_addSpellTimer4[8] = 10000        -- This timer applies to Config_addSpell4
+
+Config_addSpellModifier1bp0[8] = 3000   -- damage modifier
+Config_addSpellModifier1bp1[8] = nil    -- not required if nil
+Config_addSpellModifier2bp0[8] = nil    -- not required if nil
+Config_addSpellModifier2bp1[8] = nil    -- not required if nil
+Config_addSpellModifier3bp0[8] = nil    -- not required if nil
+Config_addSpellModifier3bp1[8] = nil    -- not required if nil
+
+Config_aura1Add1[8] = nil               -- an aura to add to the 1st add
+Config_aura2Add1[8] = nil               -- another aura to add to the 1st ad
+Config_aura1Add2[8] = nil               -- an aura to add to the 2nd add
+Config_aura2Add2[8] = nil               -- another aura to add to the 2nd add
+Config_aura1Add3[8] = nil               -- an aura to add to all ads from the 3rd on
+Config_aura2Add3[8] = nil               -- another aura to add to all add from the 3rd on
+
+Config_addSpell4Yell[8] = "Bathe in the shadows... master!"            -- yell for the adds when Spell 4 is cast
```

Comment:
```suggestion
Config_addSpell3Yell[8] = "Bathe in the shadows... master!"            -- yell for the adds when Spell 3 is cast
```
There's no system to send a yell for spell 4. We need to stick with spell 3 for this one.